CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -1223,15 +1223,33 @@ end
 --| End Gamepad Functions |--
 -----------------------------
 
+local function EvaluateBackpackPanelVisibility(enabled)
+	if not enabled then
```

Comment:
Feel likes we should also check StarterGui:GetCoreGuiEnabled(Enum.CoreGuiType.Backpack)